tests/cin_census/test_file_creator.py

```diff
@@ -6,7 +6,7 @@
 
 
 def test_get_year():
-    input = r"test\path\cin-2022.xml"
+    year = r"2022"
     la_log_dir = tmp.gettempdir()
```

Comment:
I would remove line 10 as we don't use this anymore. You can also remove the `r` in front of `r"2022` as this just turns it into a raw string which isn't necessary anymore. Raw strings are commonly used when referring to file paths so that Python doesn't interpret the slashes as commands e.g. \n for new line

Reply:
Done